src/example/App/index.js

```diff
@@ -80,26 +82,63 @@ const ComponentLinks = React.createClass({
 
   componentWillUnmount() {
     store.dispatch(actions.removeParam('component'));
+    store.dispatch(actions.removeParam('sorting'));
   },
 
 
   render() {
     const {routingState} = this.props;
 
+    const sortIt = (original, direction) => {
```

Comment:
I think it might be better to add another example for this instead of modifying existing one, so each separate example stays lean.